.do/deploy.template.yaml

```diff
@@ -0,0 +1,15 @@
+---
+spec:
+  name: digitalocean-exporter
+  services:
+    - name: digitalocean-exporter
+      github:
+        repo: metalmatze/digitalocean_exporter
+        branch: master
+      http_port: 9212
+      health_check:
+        http_path: /
+      envs:
+        - key: DIGITALOCEAN_TOKEN
+          value: CHANGE_ME
+          type: SECRET
```

Comment:
I'm not really sure I actually want every cloud provider to start adding these files to each repository...
Is it possible to make it work without the need for this extra file?

Reply:
I'd make an exception here as this exporter is directly related to DigitalOcean though. Just funny.

Reply:
Unfortunately, this seems to be the only supported method for deploying on the app platform

Reply:
[DO doc for reference](https://www.digitalocean.com/docs/app-platform/how-to/add-deploy-do-button/)